Source/WebCore/html/FeaturePolicy.cpp

```diff
@@ -298,6 +307,12 @@ FeaturePolicy FeaturePolicy::parse(Document& document, const HTMLIFrameElement&
             continue;
         }
 #endif
+        constexpr auto privateTokenToken { "private-token"_s };
+        if (item.startsWith(privateTokenToken)) {
+            isPrivateTokenInitialized = true;
+            updateList(document, iframe, policy.m_privateTokenRule, item.substring(privateTokenToken.length()));
+            continue;
+        }
     }
 
     // By default, camera, microphone, speaker-selection, display-capture, fullscreen, xr-spatial-tracking, screen-wake-lock, and web-share policy is 'self'.
```

Comment:
Should we update this comment for accuracy? Or perhaps we can document it through code instead by having

```auto self = document.securityOrigin().data()```

Reply:
Good idea, I'll clean up the other entries in a followup.